spec/models/refinery/jobs/settings_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+module Refinery
+  module Jobs
+    describe Setting, type: :model do
+      context ".notification_recipients=" do
+        it "delegates to Refinery::Setting#set" do
+          expect(Refinery::Setting).to receive(:set).
+            with(:job_application_notification_recipients, {
+              value: 'some value',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.